src/js/components/Grommet/doc.js

```diff
@@ -25,6 +25,14 @@ export const doc = Grommet => {
         control scrolling.`,
       )
       .defaultValue(false),
+    options: PropTypes.object
+      .description(
+        `Additional configurations that will be applied throughout the 
+        application. If options.layer.singleId === true, Layer id will be 
+        placed on the outermost Layer div and '{id}__container' will be placed 
+        on the div that creates the visible portion of the Layer.`,
```

Comment:
Additional component configurations that will be applied throughout the application. For backwards compatibility, some aspects of a component, such as id placement, cannot be changed within all of Grommet. This prop provides users with the ability to opt-in to newer functionality. If layer.singleId = true, Layer id will be placed on a single div that wraps all the Layer elements.

Reply:
options is a way for the Layer to assign a unique id to a single DOM node. currently, it supports only unique id for Layer.